modules/installation-aws-config-yaml.adoc

```diff
@@ -199,6 +205,9 @@ For production {product-title} clusters on which you want to perform installatio
 ifdef::private[]
 <12> How to publish the user-facing endpoints of your cluster. Set `publish` to `Internal` to deploy a private cluster, which cannot be accessed from the Internet. The default value is `External`.
 endif::private[]
+ifdef::gov[]
+<13> The custom CA certificate. This is required when deploying to the AWS C2S Secret Region, since the AWS API requires a custom CA trust bundle.
```

Comment:
```suggestion
<13> The custom CA certificate. This value is required when deploying to the AWS C2S Secret Region because the AWS API requires a custom CA trust bundle.
```